samples/snippets/fsharp/azure/queue-storage.fsx

```diff
@@ -12,102 +11,93 @@ let storageConnString = "..." // fill this in from your storage account
 let storageConnString = 
     CloudConfigurationManager.GetSetting("StorageConnectionString")
 *)
-//
-// Parse the connection string.
-//
-
-// Parse the connection string and return a reference to the storage account.
-let storageAccount = CloudStorageAccount.Parse(storageConnString)
 
 //
 // Create the Queue Service client.
 //
 
-let queueClient = storageAccount.CreateCloudQueueClient()
+// Instantiate a QueueClient which will be used to create and manipulate the queue
```

Comment:
It seems that the first comment after the 3-line header block for each section just repeats what the header mentions.   We should probably consider collapsing them; they're pretty redundant.